cloud/scope/powervs_cluster_test.go

```diff
@@ -23,11 +23,13 @@ import (
 
 	"github.com/IBM/go-sdk-core/v5/core"
 	"github.com/IBM/platform-services-go-sdk/resourcecontrollerv2"
+	"github.com/IBM/vpc-go-sdk/vpcv1"
 	"go.uber.org/mock/gomock"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/ptr"
 
+	"sigs.k8s.io/cluster-api-provider-ibmcloud/api/v1beta2"
```

Comment:
I think you should use the existing import `infrav1beta2 "sigs.k8s.io/cluster-api-provider-ibmcloud/api/v1beta2" `

Reply:
Used existing import